.github/workflows/test-recent-optimism-block.yml

```diff
@@ -7,8 +7,12 @@ on:
 
 jobs:
   test-recent-op-block:
-    name: Test recent OP block
+    name: Test recent OP block (${{ matrix.network }})
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        network: [opt-mainnet, base-mainnet]
```

Comment:
Nit:
```suggestion
        network: [op-mainnet, base-mainnet]
```

Reply:
This has to be `opt-mainnet` in order to get the right Alchemy URL:
https://github.com/NomicFoundation/edr/blob/1504c5c72c2b0300481d8bc1d5d52b4d0b7cdbe7/.github/workflows/test-recent-optimism-block.yml#L32

See this line in `replay-block.rs` that I forgot to remove previously as it's no longer necessary: https://github.com/NomicFoundation/edr/pull/867/commits/1504c5c72c2b0300481d8bc1d5d52b4d0b7cdbe7